UnitySDK/Assets/ML-Agents/Scripts/BrainParameters.cs

```diff
@@ -131,7 +131,7 @@ public BrainParameters(CommunicatorObjects.BrainParametersProto brainParametersP
             numStackedVectorObservations = brainParametersProto.NumStackedVectorObservations;
             vectorActionSize = brainParametersProto.VectorActionSize.ToArray();
             vectorActionDescriptions = brainParametersProto.VectorActionDescriptions.ToArray();
-            vectorActionSpaceType = (SpaceType)brainParametersProto.VectorActionSpaceType;
+            vectorActionSpaceType = (SpaceType) brainParametersProto.VectorActionSpaceType;
```

Comment:
why did the formatting change here?

Reply:
no idea. was this from your format changes @surfnerd? if not maybe i fat fingered it

Reply:
Should be no space there.